example/lib/ios_popover/_ios_toolbar.dart

```diff
@@ -0,0 +1,594 @@
+  }
+
+  @override
+  void paint(PaintingContext context, Offset offset) {
+    final children = getChildrenAsList();
+    final page = _pages![_controller.currentPage - 1];
+
+    late Offset childOffset;
+    final hasMultiplePages = (_pages?.length ?? 0) > 1;
+
+    if (hasMultiplePages) {
+      // Paint the previous page button.
+      final previousButton = children.first;
+      childOffset = (previousButton.parentData as _IosPagerParentData).offset;
+      context.paintChild(previousButton, childOffset + offset);
+    }
+
+    for (int i = page.startingIndex; i < page.endingIndex; i++) {
+      final child = children[i];
+      childOffset = (child.parentData as _IosPagerParentData).offset;
+
+      if (hasMultiplePages || i > page.startingIndex) {
+        // Paint the separator.
+        context.canvas.drawLine(
+          offset + Offset(childOffset.dx, 0),
+          offset + Offset(childOffset.dx, size.height),
+          _linePaint,
+        );
+      }
+
+      // Paint the child content.
+      context.paintChild(child, childOffset + offset);
+    }
+
+    if (hasMultiplePages) {
+      final nextButton = children.last;
+      childOffset = (nextButton.parentData as _IosPagerParentData).offset;
+
+      // Paint the separator.
+      context.canvas.drawLine(
+        offset + Offset(childOffset.dx, 0),
+        offset + Offset(childOffset.dx, size.height),
+        _linePaint,
+      );
+
+      // Paint the next page button.
+      context.paintChild(nextButton, childOffset + offset);
+    }
+  }
+
+  @override
+  bool hitTestChildren(BoxHitTestResult result, {required Offset position}) {
+    final page = _pages![_controller.currentPage - 1];
+
+    final children = getChildrenAsList();
+    final previousButton = children.first;
+    final nextButton = children.last;
+
+    // Check if we hit the previous button.
+    if (_hitTestChild(result, position: position, child: previousButton)) {
+      return true;
+    }
+
+    // Check if we hit the next button.
+    if (_hitTestChild(result, position: position, child: nextButton)) {
+      return true;
+    }
+
+    // Hit test the items on the current page.
+    for (int i = page.startingIndex; i < page.endingIndex; i++) {
+      final child = children[i];
+      final isHit = _hitTestChild(
+        result,
+        position: position,
+        child: child,
+      );
+      if (isHit) {
+        return true;
+      }
+    }
+
+    return false;
+  }
+
+  bool _hitTestChild(BoxHitTestResult result, {required Offset position, required RenderBox child}) {
+    final childParentData = child.parentData! as _IosPagerParentData;
+
+    return result.addWithPaintOffset(
+      offset: childParentData.offset,
+      position: position,
+      hitTest: (BoxHitTestResult result, Offset transformed) {
+        assert(transformed == position - childParentData.offset);
+        return child.hitTest(result, position: transformed);
+      },
+    );
+  }
+
+  /// Computes all the pages.
```

Comment:
This restates the name of the method. What does it mean to "compute all pages"?

Reply:
Updated.